### ISSUE TITLE
0 byte, 0 second output file generated (comfyui v0.3.43)

### ISSUE THREAD

**Issue by BaseMe2** [filliptm/ComfyUI_Fill-ChatterBox#21]
Console only shows:

> contextlib.py:105: FutureWarning: `torch.backends.cuda.sdp_kernel()` is deprecated. In the future, this context manager will be removed. Please see `torch.nn.attention.sdpa_kernel()` for the new context manager, with updated signature.

Works fine in ComfyUI 0.3.38. Both portable edition. I did redownload chatterbox, just in case. 

>absl-py                 2.2.2
accelerate              1.6.0
aiofiles                24.1.0
albucore                0.0.24
albumentations          2.0.8
annotated-types         0.7.0
antlr4-python3-runtime  4.9.3
anyio                   4.9.0
audioread               3.0.1
av                      14.0.1
bitsandbytes            0.45.4
certifi                 2025.4.26
cffi                    1.17.1
cfgv                    3.4.0
charset-normalizer      3.4.1
click                   8.1.8
cloudpickle             3.1.1
colorama                0.4.6
coloredlogs             15.0.1
colorlog                6.9.0
conformer               0.3.2
contourpy               1.3.2
cycler                  0.12.1
Cython                  3.0.12
decorator               5.2.1
diffusers               0.33.1
distlib                 0.3.9
easydict                1.13
einops                  0.7.0
facexlib                0.3.0
fastapi                 0.115.12
ffmpy                   0.5.0
filelock                3.18.0
filterpy                1.4.5
flatbuffers             25.2.10
fonttools               4.58.2
fsspec                  2025.3.2
ftfy                    6.3.1
gitdb                   4.0.12
GitPython               3.1.44
gradio                  5.29.0
gradio_client           1.10.0
groovy                  0.1.2
grpcio                  1.71.0
h11                     0.16.0
httpcore                1.0.9
httpx                   0.28.1
huggingface-hub         0.30.0
humanfriendly           10.0
hydra-colorlog          1.2.0
hydra-core              1.3.2
identify                2.6.12
idna                    3.10
imageio                 2.37.0
importlib_metadata      8.7.0
insightface             0.7.3
Jinja2                  3.1.6
joblib                  1.4.2
kiwisolver              1.4.8
lazy_loader             0.4
librosa                 0.11.0
llvmlite                0.44.0
Markdown                3.8
markdown-it-py          3.0.0
MarkupSafe              3.0.2
matplotlib              3.10.3
mdurl                   0.1.2
mmaudio                 1.0.0                    D:\AI\comfyUI\ComfyUI\custom_nodes\MMAudio
mpmath                  1.3.0
msgpack                 1.1.0
networkx                3.4.2
nitrous_ema             0.0.1
nodeenv                 1.9.1
numba                   0.61.2
numpy                   2.0.2
omegaconf               2.3.0
onnx                    1.18.0
onnxruntime             1.22.0
open_clip_torch         2.32.0
opencv-python           4.10.0.84
opencv-python-headless  4.11.0.86
orjson                  3.10.18
packaging               25.0
pandas                  2.2.3
pillow                  11.2.1
pip                     25.1.1
platformdirs            4.3.7
polygraphy              0.49.20
pooch                   1.8.2
pre_commit              4.2.0
prettytable             3.16.0
protobuf                6.30.2
psutil                  7.0.0
pycparser               2.22
pydantic                2.11.4
pydantic_core           2.33.2
pydub                   0.25.1
Pygments                2.19.1
pyparsing               3.2.3
pyreadline3             3.5.4
python-dateutil         2.9.0.post0
python-dotenv           1.1.0
python-multipart        0.0.20
pytz                    2025.2
PyYAML                  6.0.2
regex                   2024.11.6
requests                2.32.3
resampy                 0.4.3
rich                    14.0.0
ruff                    0.11.8
s3tokenizer             0.1.7
safehttpx               0.1.6
safetensors             0.4.5
scikit-image            0.25.2
scikit-learn            1.6.1
scipy                   1.15.2
semantic-version        2.10.0
sentencepiece           0.2.0
setuptools              80.1.0
shellingham             1.5.4
simsimd                 6.4.9
six                     1.17.0
smmap                   5.0.2
sniffio                 1.3.1
soundfile               0.13.1
soxr                    0.5.0.post1
starlette               0.46.2
stringzilla             3.12.5
sympy                   1.14.0
tensorboard             2.19.0
tensorboard-data-server 0.7.2
tensordict              0.8.1
tensorrt                10.10.0.31
threadpoolctl           3.6.0
tifffile                2025.6.1
timm                    1.0.15
tokenizers              0.20.3
toml                    0.10.2
tomli                   2.2.1
tomlkit                 0.13.2
torch                   2.8.0.dev20250501+cu128
torchaudio              2.6.0.dev20250502+cu128
torchdiffeq             0.2.5
torchsde                0.2.6
torchvision             0.22.0.dev20250502+cu128
tqdm                    4.67.1
trampoline              0.1.2
transformers            4.46.3
typer                   0.15.3
typing_extensions       4.13.2
typing-inspection       0.4.0
tzdata                  2025.2
urllib3                 2.4.0
uvicorn                 0.34.2
virtualenv              20.31.2
voluptuous              0.15.2
wcwidth                 0.2.13
websockets              15.0.1
Werkzeug                3.1.3
zipp                    3.21.0

**Comment by xyzConfig** [comment]
same here..

**Comment by moviemakr1620** [comment]
same

**Comment by millerlight** [comment]
this bugfix works for me:
https://github.com/filliptm/ComfyUI_Fill-ChatterBox/issues/24
